snippets/distr-tracing-tempo-tempomonolithic-custom-resource.adoc

```diff
@@ -19,14 +19,23 @@ spec:
       size: <value>Gi # <2>
       s3: # <3>
         secret: <secret_name> # <4>
+    tls:
+      caName: <ca_certificate_configmap_name> # <5>
   jaegerui:
-    enabled: true # <5>
+    enabled: true # <6>
     route:
-      enabled: true # <6>
+      enabled: true # <7>
+  resources: # <8>
+    total:
+      limits:
+        memory: <value>Gi
+        cpu: <value>m
 ----
-<1> Type of storage for storing traces: in-memory storage, a persistent volume, or object storage. The value for the `tmpfs` in-memory storage is `memory`. The value for a persistent volume is `pv`. The accepted values for object storage are `s3`, `gcs`, or `azure`, depending on the used object store type.
+<1> Type of storage for storing traces: in-memory storage, a persistent volume, or object storage. The value for a persistent volume is `pv`. The accepted values for object storage are `s3`, `gcs`, or `azure`, depending on the used object store type. The default value is `memory` for the `tmpfs` in-memory storage, which is only appropriate for development, testing, demonstrations, and proof-of-concept environments because the data does not persist when the pod is shut down.
 <2> Memory size: For in-memory storage, this means the size of the `tmpfs` volume, where the default is `2Gi`. For a persistent volume, this means the size of the persistent volume claim, where the default is `10Gi`. For object storage, this means the size of the persistent volume claim for the Tempo WAL, where the default is `10Gi`.
 <3> Optional: For object storage, the type of object storage. The accepted values are `s3`, `gcs`, and `azure`, depending on the used object store type.
 <4> Optional: For object storage, the value of the `name` in the `metadata` of the storage secret. The storage secret must be in the same namespace as the TempoMonolithic instance and contain the fields specified in "Table 1. Required secret parameters" in the section "Object storage setup".
-<5> Enables the Jaeger UI.
-<6> Enables creation of a route for the Jaeger UI.
+<5> Optional: Name of a `ConfigMap` object that contains a CA certificate.
+<6> Enables the Jaeger UI.
+<7> Enables creation of a route for the Jaeger UI.
+<8> Optional?
```

Comment:
Is the `resources` section optional?

Reply:
Yeah, resources is optional. We can create the Tempo monolithic instance without specifying the resources.